common/sys/intrinsics.h

```diff
@@ -89,7 +89,7 @@ namespace embree
 #endif
   }
   
-#if defined(__X86_64__)
+#if defined(__X86_64__) || defined (__aarch64__)
```

Comment:
`__aarch64__` is not defined on a pure Arm64 Windows environment with Arm64 MSVC, so this change is not sufficient to make it compile. However, `_M_ARM64` is defined.

Why is this function conditionally compiled for only x86_64 and arm64 anyway? Can we just replaced it with a `uint64_t` version and make it always compiled? Or can we make it compiled for any 64-bit platform?